deploy-manage/deploy/cloud-on-k8s/install-using-yaml-manifest-quickstart.md

```diff
@@ -14,5 +17,79 @@ mapped_urls:
+
+* If you are using Amazon EKS, make sure the Kubernetes control plane is allowed to communicate with the Kubernetes nodes on port 443. This is required for communication with the Validating Webhook. For more information, check [Recommended inbound traffic](https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.md).
+
+* Refer to [*Install ECK*](../../../deploy-manage/deploy/cloud-on-k8s/install.md) for more information on installation options.
+
+* Check the [upgrade notes](../../../deploy-manage/upgrade/orchestrator/upgrade-cloud-on-k8s.md) if you are attempting to upgrade an existing ECK deployment.
+
+##  Installation procedure
+
+To deploy the ECK operator:
+
+1. Install [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) with [`create`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/):
+
+    ```sh
+    kubectl create -f https://download.elastic.co/downloads/eck/{{eck_version}}/crds.yaml
+    ```
+
+    This will output similar to the following upon Elastic resources' creation:
+
+    ```sh
+    customresourcedefinition.apiextensions.k8s.io/agents.agent.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/apmservers.apm.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/beats.beat.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/elasticmapsservers.maps.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/elasticsearches.elasticsearch.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/enterprisesearches.enterprisesearch.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/kibanas.kibana.k8s.elastic.co created
+    customresourcedefinition.apiextensions.k8s.io/logstashes.logstash.k8s.elastic.co created
+    ```
+
+2. Install the operator with its RBAC rules with [`apply`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_apply/):
+
+    ```sh
+    kubectl apply -f https://download.elastic.co/downloads/eck/{{eck_version}}/operator.yaml
+    ```
+
+    ::::{note}
+    The ECK operator runs by default in the `elastic-system` namespace. It is recommended that you choose a dedicated namespace for your workloads (such as Elasticsearch and Kibana), rather than using the `elastic-system` or the `default` namespace.
+    ::::
+
+3. Monitor the operator’s setup from its logs through [`logs`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_logs/):
+
+    ```sh
+    kubectl -n elastic-system logs -f statefulset.apps/elastic-operator
+    ```
+
+4. Once ready, the operator will report as `Running` as shown with [`get`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/), replacing default `elastic-system` with applicable installation namespace as needed: *
```

Comment:
```suggestion
4. Use [`kubectl get`](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_get/) to check the operator status, passing in the namespace using the `-n` flag. When the operator is ready to use, it will report as `Running`. 
```